proposals/infra/INF-0002-root-signature-in-clang.md

```diff
@@ -151,6 +151,165 @@ We have received feedback that the DSL for root signatures is not something that
 every language that targets DirectX would want to adopt. For this reason we need
 to ensure that our solution doesn't unnecessarily tie the non-HLSL parts to it.
 
+### Root Signature Grammar
```

Comment:
Is there are public reference we could link to for the grammar?

Reply:
Cannot find reference. And most things are for fxc which doesn't include new flags added for newer shader models.